### DISCUSSION TITLE
A large number of values are smaller than 1e-05. For a self-join, try setting `ignore_trivial = True`.

### DISCUSSION THREAD

**Discussion by adityabhandwalkar** [TDAmeritrade/stumpy#696] [General]
Hello I am consistently getting this msg on my console while I am trying  to run my code for mpdist and whenever I add 

   `     distance = stumpy.mpdist(X[i], X[j], int(window_size),ignore_trivial=True)`

I encounter this error msg 
` line 132, in inner_wrapper return norm(*args, **kwargs) TypeError: mpdist() got an unexpected keyword argument 'ignore_trivial'`

I need help with this

**Comment by seanlaw** [comment]
> A large number of values are smaller than 1e-05. For a self-join, try setting ignore_trivial = True

@adityabhandwalkar In this case, you can likely ignore this warning (this is not an error) and leave out `ignore_trivial=True` as it doesn't do anything. This is sufficient:

```
distance = stumpy.mpdist(X[i], X[j], int(window_size))
```

The only reason why this warning comes up is because `X[i]` and `X[j]` are virtually identical in many places, which is possible when using `mpdist`.
